LuaMenu/widgets/gui_campaign_handler.lua

```diff
@@ -930,33 +931,39 @@ local function SelectPlanet(popupOverlay, planetHandler, planetID, planetData, s
 	end
 
 	local function SizeUpdate()
-		local font = Configuration:GetFont(((planetHandler.height < 760) and 2) or 3)
+		local fluffFont = Configuration:GetFont(((planetHandler.height < 720) and 2) or 3)
+		local descFont = Configuration:GetFont(((planetHandler.height < 720) and 1) or 2) 
 
-		planetDesc.font.size = font.size
+		planetDesc.font.size = descFont.size
 		planetDesc:Invalidate()
-		if planetHandler.height < 660 then
+		if planetHandler.height < 560 then
 			planetDesc._relativeBounds.top = 60
 			fluffGrid:SetVisibility(false)
+
 		elseif planetHandler.height < 820 then
-			planetDesc._relativeBounds.top = "27%"
+			planetDesc._relativeBounds.top = "26%"
 			fluffGrid:SetVisibility(true)
+
 		else
-			planetDesc._relativeBounds.top = "30%"
+			planetDesc._relativeBounds.top = "25%"
 			fluffGrid:SetVisibility(true)
 		end
 		planetDesc:UpdateClientArea(false)
 
-		if planetHandler.height < 600 then
-			subPanel._relativeBounds.right = 390
-			subPanel._relativeBounds.bottom = "2%"
-		else
+		if planetHandler.height > 800 then
```

Comment:
This would be more resilient if it depended on width.

Reply:
Agreed, it does seem a strange decision to be using height. All of the resize calls (including non-width ones that existed before) have been relying on height as their primary breakpoints so I just kept the convention. Is it better to intermix changes to both width and height? There does not appear to be a direct mapping to screen resolution in planetHandler that I know of so the numbers may still end up a bit arbitrary.